src/git_auto_export/README.md

```diff
@@ -0,0 +1,57 @@
+#### Steps to install on edX platform:
+
+
+You can install this plugin into any Open edX instance by using any of the following methods:
+
+
+**Option 1: Install from PyPI**
+
+.. code-block::
+
+    # If running devstack in docker, first open a shell in LMS (make lms-shell)
+
+    pip install ol-openedx-canvas-integration
+
+
+**Option 2: Build the package locally and install it**
+
+Follow these steps in a terminal on your machine:
+
+1. Navigate to ``open-edx-plugins`` directory
+2. If you haven't done so already, run ``./pants build``
+3. Run ``./pants package ::``. This will create a "dist" directory inside "open-edx-plugins" directory with ".whl" & ".tar.gz" format packages for all the "ol_openedx_*" plugins in "open-edx-plugins/src")
+4. Move/copy any of the ".whl" or ".tar.gz" files for this plugin that were generated in the above step to the machine/container running Open edX (NOTE: If running devstack via Docker, you can use ``docker cp`` to copy these files into your LMS container)
+5. Run a shell in the machine/container running Open edX, and install this plugin using pip
+
+
+- open cms.env and lms.env or private.py and set FEATURES flags
+
+  ```
+  "FEATURES": {
+    "ENABLE_EXPORT_GIT": true,
+    "ENABLE_GIT_AUTO_EXPORT": true
```

Comment:
I have run that functionality by setting them up in edx-platform. I guess that's why you were unable to test it because you didn't set that.

Reply:
Here are the reference that these are the part of the edx-platform

https://github.com/openedx/edx-platform/blob/4119e60fa93407e30765ff8fd7864cc8b5134819/cms/djangoapps/contentstore/toggles.py#L6-19

<img width="1440" alt="Screenshot 2022-04-26 at 15 05 08" src="https://user-images.githubusercontent.com/4043989/165276501-2963e857-a774-4cea-935d-66d1f4713853.png">
<img width="1440" alt="Screenshot 2022-04-26 at 15 05 25" src="https://user-images.githubusercontent.com/4043989/165276530-62b2db09-6af2-48a4-941b-18cd079ff4e6.png">

Reply:
The other one exists in the platform, I was talking about `ENABLE_GIT_AUTO_EXPORT` specifically. Also, in your screenshot `ENABLE_GIT_AUTO_EXPORT` is added manually as we can see.

Setting this up in `private.py` will give you the value even though that's not part of the platform itself but the good practice is to add the settings as mentioned in above comment so that the your plugin introduces the settings that is only needed by your plugin and which is not part of edx-platform core.